numpy/ma/core.py

```diff
@@ -4343,7 +4344,7 @@ def imag(self):
         Return the imaginary part of the masked array.
```

Comment:
As part of making these properties, you should try to make the docstring describe the setting and getting at the same time. In this case, drop the "return" here, and change the sentence below to "when accessed, the returned array is ..."

Reply:
Do you have any examples of how the documentation should look for a property to handle both getter and setter? For instance, is it good to leave the "Parameters" section and list "values" or is this a special case since it's a setter and there can only be one value?

Reply:
Looking at [the doc docs](https://numpydoc.readthedocs.io/en/latest/format.html), I can't actually find any guidelines on documenting properties.

I'd say the `Parameters` header should go, as should the return value section. `help(np.ndarray.imag)` is an existing example, as is `help(np.ndarray.flat)`